README.asciidoc

```diff
@@ -1,2 +1,7 @@
-TODO
-Instructions how to set up and run the code.
+== Instructions
+
+1. Get the required libraries.
+
+2. Run motion.py to test if motors work.
+
+3. Run main.py to see the robot in action.
```

Comment:
Are aspects of the code configurable? How to configure them?

Reply:
described, how to configure the code